site/_data/fuguApis.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = async () => {
+  const fuguFile = path.join(
+    __dirname,
+    '../../external/data/fugu-showcase.json'
+  );
+  const fuguItems = /** @type {FuguProject[]} */ (
+    JSON.parse(fs.readFileSync(fuguFile, 'utf-8'))
```

Comment:
As this method is `async` anyway consider using `fs/promises` instead of `readFileSync`.